tests/helics/application_api/MessageFederateAdditionalTests.cpp

```diff
@@ -326,12 +326,12 @@ class PingPongFed
     std::string name;  //!< the name of the federate
     helics::core_type coreType;
     std::vector<std::pair<helics::Time, std::string>> triggers;
-    helics::Endpoint *ep;
-    int index = 0;
+    helics::Endpoint *ep{nullptr};
+    int index{0};
 
   public:
-    int pings = 0;  //!< the number of pings received
-    int pongs = 0;  //!< the number of pongs received
+    int pings{0};  //!< the number of pings received
```

Comment:
not really for plain ints,  shouldn't make any difference,  just making it consistent,  there are other types where it could make a difference, then brace initialization is slightly preferred or in the case of atomics the only way.    I have just been moving more toward using brace initialization more, so I just changed it.